paywall/next.config.js

```diff
@@ -5,13 +5,23 @@ const withSourceMaps = require('@zeit/next-source-maps')
 
 const copyFile = promisify(fs.copyFile)
 
+const publicRuntimeConfig = {
```

Comment:
we can get around the test coverage failure by taking this line only outside:

```js
const unlockEnv = process.env.UNLOCK_ENV || 'dev'
```

and then doing:
```js
publicRuntimeConfig: {
  unlockEnv,
  unlockAddress: process.env.UNLOCK_ADDRESS || (unlockEnv === 'dev' && '0x885EF47c3439ADE0CB9b33a4D3c534C99964Db93')
```